### ISSUE TITLE
About pre-trained model for fine-tuning

### ISSUE THREAD

**Issue by ChenCong7375** [AlexeyAB/darknet#3283]
Dear Alexey,
     Now I want to fine-tune on my little dataset,but I designed my net by myself.
     So how should I get pre-trained model? Should I train my net on imagenet1k or VOC07+12 then partial the weight using command line? 
     And where can I find [imagenet1k.train.list] and [imagenet1k.valid.list]?

**Comment by AlexeyAB** [comment]
@ChenCong7375 Hi,

>  I designed my net by myself.

You should create classification network that has the same first several layers as in your detection net.
Then train this classification net on Imagenet.
Then do partial for the same layers.

**Comment by ChenCong7375** [comment]
@AlexeyAB Morning,
Could you please tell me how long did you train your nets?
About one week?
I have a Titan Xp and i7-8700k.

**Comment by ChenCong7375** [comment]
@AlexeyAB Moreover, How can I get [imagenet1k.train.list] and [imagenet1k.valid.list]?
Should I create it?
I downloaded the  ILSVRC2012 but I am at a loss as to what to do.

**Comment by AlexeyAB** [comment]
> About one week?

Yes, more or less.

If you already downloaded `ILSVRC2012_img_train.tar` then un-comment all lines except `wget` in this script and run it: https://github.com/AlexeyAB/darknet/blob/master/scripts/windows/windows_imagenet_train.sh

And train the classifier on ILSVRC2012: 
`./darknet classifier train cfg/imagenet1k.data cfg/darknet53_448_xnor.cfg darknet53_xnor.conv.74  -topk`

**Comment by iaixuexi** [comment]
Whether to need classification pre-training on imagenet after changing the network structure?
thank you
